<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HornApp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="buttonsLayout">
      <item>
       <widget class="QPushButton" name="newPostButton">
        <property name="text">
         <string>Создать пост</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="refreshFeedButton">
        <property name="text">
         <string>Обновить ленту</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>355</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFeed">
    <property name="title">
     <string>Лента</string>
    </property>
    <addaction name="actionNewPost"/>
    <addaction name="actionRefreshFeed"/>
    <addaction name="actionOpenPost"/>
    <addaction name="separator"/>
    <addaction name="actionNotifications"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFeed"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNewPost">
   <property name="text">
    <string>Создать пост…</string>
   </property>
   <property name="toolTip">
    <string>Создать пост</string>
   </property>
  </action>
  <action name="actionRefreshFeed">
   <property name="text">
    <string>Обновить ленту</string>
   </property>
   <property name="toolTip">
    <string>Обновить ленту</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>О Qt</string>
   </property>
   <property name="toolTip">
    <string>О Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionNotifications">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Уведомления</string>
   </property>
   <property name="toolTip">
    <string>Уведомления</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+N</string>
   </property>
  </action>
  <action name="actionOpenPost">
   <property name="text">
    <string>Открыть пост…</string>
   </property>
   <property name="toolTip">
    <string>Открыть пост</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
